h2o-logger/src/main/java/logging/ConsoleLogger.java

```diff
@@ -0,0 +1,43 @@
+package logging;
```

Comment:
should be either water, (public) or hex. (internal)

Reply:
fixed